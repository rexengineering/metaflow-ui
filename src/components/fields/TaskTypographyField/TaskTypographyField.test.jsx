import { render, screen } from "@testing-library/react";
import React from "react";
import TaskTypographyField from ".";

describe("<TaskTypographyField />", () => {
  it("should render", () => {
    const { asFragment } = render(<TaskTypographyField />);
    const initialRender = asFragment();

    expect(initialRender).not.toBeNull();
  });

  it("should match snapshot", () => {
    const { asFragment } = render(<TaskTypographyField data="sometext" />);
    const initialRender = asFragment();

    expect(initialRender).toMatchSnapshot();
  });

  it("should render the supplies text", () => {
    render(<TaskTypographyField data="sometext" />);

    const element = screen.getByText("sometext");
    expect(element).toBeInTheDocument();
  });
});
