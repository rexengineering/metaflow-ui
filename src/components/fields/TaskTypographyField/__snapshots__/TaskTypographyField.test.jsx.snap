// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TaskTypographyField /> should match snapshot 1`] = `
<DocumentFragment>
  <p
    class="MuiTypography-root MuiTypography-body1"
  >
    sometext
  </p>
</DocumentFragment>
`;
